### DISCUSSION TITLE
`send-text` with string that starts with '-'

### DISCUSSION THREAD

**Discussion by jghauser** [kovidgoyal/kitty#5486] [Q&A] [answered]
I have a neovim [plugin](https://github.com/jghauser/kitty-runner.nvim) that allows sending text from neovim to a kitty terminal window with `send-text`. A user has opened an [issue](https://github.com/jghauser/kitty-runner.nvim/issues/7) because he's unable to send strings that start with the character '-'. I've now realised that this is because kitty thinks that '-' begins a new option rather than a standard string starting with '-'. I've not been able to escape the '-' without thereby having "" or '' characters being sent also.

Is there some way of sending strings that start with '-' or is this an inherent limitation of kitty?

Thanks! And thanks for all the great work on kitty!

**Comment by kovidgoyal** [comment, marked as answer]
kitty @ send-text -- -whatever

**Comment by kovidgoyal** [comment]
or echo "-whatever" | kitty @ send-text --stdin

**Comment by jghauser** [comment]
So simple, could really have come up with this myself :facepalm:. Thanks a lot for the super quick response!
